lib/mayaUsd/ufe/UsdContextOps.cpp

```diff
@@ -90,6 +95,33 @@ class SetVariantSelectionUndoableCommand : public Ufe::UndoableCommand
     const std::string fNewSelection;
 };
 
+//! \brief Undoable command for prim active state change
+class ToggleActiveStateCommand : public Ufe::UndoableCommand
+{
+public:
+    ToggleActiveStateCommand(const UsdPrim& prim)
+        : _prim(prim)
+    {
+        _active = _prim.IsActive();
+    }
+
+    void undo() override
+    {
+        MayaUsd::ufe::InAddOrDeleteOperation ad;
```

Comment:
Technically we are not an add or delete operation but by activate/deactivate a prim we want to send the UFE add or delete notifs.